vector/src/main/java/im/vector/app/features/html/VectorCharSequenceFactory.kt

```diff
@@ -24,20 +24,22 @@ import im.vector.app.EmojiSpanify
 import im.vector.app.features.home.room.detail.timeline.item.BindingOptions
 import javax.inject.Inject
 
-class SpanUtils @Inject constructor(
+data class VectorCharSequence(val value: CharSequence, val bindingOptions: BindingOptions)
```

Comment:
we could potentially override the hashcode/equals here to avoid epoxy throwing when spannables mutate but I chose to leave out the change from this PR